cmd/lotus/rpc.go

```diff
@@ -46,7 +46,14 @@ func serveRPC(a api.FullNode, stop node.StopFunc, addr multiaddr.Multiaddr) erro
 
 	http.Handle("/rest/v0/import", importAH)
 
-	http.Handle("/metrics", promhttp.Handler())
+	exporter, err := prometheus.NewExporter(prometheus.Options{
+		Namespace: "lotus",
+	})
+	if err != nil {
+		log.Fatalf("could not create the prometheus stats exporter: %v", err)
+	}
+
+	http.Handle("/metrics", exporter)
```

Comment:
go-ipfs exports it under `/debug/metrics`, if it isn't much of an issue, it would be easiest if we did the same

Reply:
Will do!